Jenkinsfile

```diff
@@ -13,10 +13,7 @@ pipeline {
             steps {
                 sh 'printenv'
                 checkout scm
-                script {
-                    commitHash = getCommitHash()
-                    runApprobation ignoreFailure: false, vegawalletDesktopBranch: commitHash, type: 'frontend'
-                }
+                runApprobation ignoreFailure: false, vegawalletDesktopBranch: env.BRANCH_NAME, type: 'frontend'
```

Comment:
Is something wrong with `getCommitHash()`? It uses `env.BRANCH_NAME` as one of the options.